package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "featuredfilm",
+  "version": "1.0.0",
+  "description": "Module to display featured film on Moodu",
+  "main": "index.js",
+  "scripts": {
+    "test": "jest",
+    "webpack-watch": "webpack -d --watch",
```

Comment:
I would recommend alphabetizing your scripts to avoid accidental duplication/overwriting of previous key-value pairs in an object.